File: mycollab-mobile/src/main/scala/com/mycollab/mobile/module/project/events/ProjectEvent.scala
package com.mycollab.mobile.module.project.events

import com.mycollab.events.ApplicationEvent

/**
  * @author MyCollab Ltd
  * @since 5.2.5
  */
object ProjectEvent {

  class GotoAdd(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoProjectList(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoMyProject(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class MyProjectActivities(source: AnyRef, data: Integer) extends ApplicationEvent(source, data) {}

  class GotoAllActivitiesView(source: AnyRef) extends ApplicationEvent(source, null) {}

  class GotoDashboard(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

}

File: mycollab-mobile/src/main/java/com/mycollab/mobile/module/project/event/ProjectEvent.kt
package com.mycollab.mobile.module.project.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
object ProjectEvent {
    class GotoAdd(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class GotoProjectList(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class GotoMyProject(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class MyProjectActivities(source: Any, data: Int) : ApplicationEvent(source, data) 

    class GotoAllActivitiesView(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class GotoDashboard(source: Any, data: Any?) : ApplicationEvent(source, data) 
}